Assets/_BForBoss/_Utility/Scripts/CustomPassVolumeWeightTool.cs

```diff
@@ -37,12 +37,12 @@ public CustomPassVolumeWeightTool(CustomPassVolume customPassVolume, string mate
         
         public Tweener Distort()
         {
-            return DOTween.To(intensity => _material.SetFloat(_materialKey, intensity), _startValue, _endValue, _distortDuration);
```

Comment:
This ensures that the tweens use unscaled time. Otherwise, the blackscreen never shows up and the tween never completes